pkg/antctl/command_definition.go

```diff
@@ -320,6 +327,209 @@ func respTransformer(obj interface{}) (interface{}, error) {
 	return target, nil
 }
 
+// getTableHeaders get the headers of the output table.
+func (cd *commandDefinition) getTableHeaders(target interface{}, tab *table.Writer) ([]string, error) {
+	args := make([]string, 0)
+	headers, ok := target.(map[string]interface{})
+	if !ok {
+		return nil, errors.New("converting types error in getTableHeaders")
+	}
+
+	// ensure that "name" column is the first column if it exists
+	// sort other columns in alphabet order in default
+	hasNameHeader := false
+	nameHeader := ""
+	tabConfigs := make([]table.ColumnConfig, 0)
+	for header := range headers {
+		if strings.ToLower(header) == "name" {
+			hasNameHeader = true
+			nameHeader = header
+			continue
+		}
+		args = append(args, header)
+	}
+	sort.Strings(args)
+	if hasNameHeader {
+		args = append([]string{nameHeader}, args...)
+	}
+
+	tabHeaders := table.Row{}
+	// columnNameMap stores column names that are needed to be converted in text
+	columnNameMap := map[string]string{
+		"appliedtogroups": "applied-to",
+	}
```

Comment:
Not good to do non-generic conversion in a generic function, maybe just capitalize it too.